docker/nginx/conf.d/default.dev.conf

```diff
@@ -2,43 +2,23 @@ server {
     root /srv/sylius/public;
 
     location / {
-        # try to serve file directly, fallback to app.php
-        try_files $uri /app.php$is_args$args;
+        # try to serve file directly, fallback to index.php
+        try_files $uri /index.php$is_args$args;
     }
 
     # DEV
     # This rule should only be placed on your development environment
     # In production, don't include this and don't deploy app_dev.php or config.php
-    location ~ ^/(app_dev|config)\.php(/|$) {
-        # Comment the next line and uncomment the lines after to enable dynamic resolution (incompatible with Kubernetes)
+    location ~ ^/index\.php(/|$) {
         fastcgi_pass php:9000;
-        #resolver 127.0.0.11;
-        #set $upstream_host php;
-        #fastcgi_pass $upstream_host:9000;
-
         fastcgi_split_path_info ^(.+\.php)(/.*)$;
         include fastcgi_params;
-        # When you are using symlinks to link the document root to the
```

Comment:
Same comment here. When you are dealing with some deployment methods, the application root is often a symlink to the latest version. So I would keep this lines commented and/or put a link to a documentation which explains everything about.